project/Settings.scala

```diff
@@ -0,0 +1,83 @@
+import sbt._
+
+/**
+ * Application settings and dependencies
+ */
+object Settings {
+  object Definitions {
+    /** The name of the application */
+    val name = "ocs3"
+
+    /** Project version */
+    val version = "2016001.1.1"
```

Comment:
@swalker2m yeah I have similar thoughts on the version thing. I suggest we start with `0.0.1-SNAPSHOT` like normal software projects and see how that goes. The fewer weird things we have from the outset the better off we will be I think.